tailwind.config.ts

```diff
@@ -48,6 +48,9 @@ module.exports = {
 					DEFAULT: "hsl(var(--card))",
 					foreground: "hsl(var(--card-foreground))",
 				},
+				"parla-icon-grey": "#64748B",
```

Comment:
I would call the colors
- parla-blue
- parla-grey
- and parla-black

since I am also using them for the button background and not only icons

Reply:
sounds good! 👍